packages/cli/src/utils.ts

```diff
@@ -197,9 +197,9 @@ const validatePath = async (value: string) => {
   return true;
 };
 
-export const inquireInstancePath = async (initialPath?: string) => {
+export const inquireInstancePath = async (initialPath?: string, skipValidation?: boolean) => {
   const inquire = async () => {
-    if (!initialPath && (await validatePath('.')) === true) {
+    if (!initialPath && ((await validatePath('.')) === true || skipValidation)) {
```

Comment:
`skipValidation || (await validatePath('.')) === true`
if the `skipValidation` is true then no need to check the value of `validatePath`?

Reply:
Good point